app/src/main/java/com/brainwallet/tools/qrcode/QRUtils.java

```diff
@@ -83,6 +83,21 @@ public static boolean generateQR(Context ctx, String bitcoinURL, ImageView qrcod
         return true;
     }
 
+    public static Bitmap generateQR(Context ctx, String bitcoinURL) {
```

Comment:
We should use litecoin naming where ever possible.  I know Breadwallet was for Bitcoin we just never prioritized changed the names....but in this case it seems a good idea

Reply:
noted!